src/Sylius/Bundle/GridBundle/spec/Templating/Helper/GridHelperSpec.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\GridBundle\Templating\Helper;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Bundle\GridBundle\Templating\Helper\GridHelper;
+use Sylius\Component\Grid\Definition\Action;
+use Sylius\Component\Grid\Definition\Field;
+use Sylius\Component\Grid\Renderer\GridRendererInterface;
+use Sylius\Component\Grid\View\GridView;
+use Symfony\Component\Templating\Helper\Helper;
+use Symfony\Component\Templating\Helper\HelperInterface;
+
+/**
+ * @mixin GridHelper
+ *
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class GridHelperSpec extends ObjectBehavior
+{
+    function let(GridRendererInterface $gridRenderer)
+    {
+        $this->beConstructedWith($gridRenderer);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\GridBundle\Templating\Helper\GridHelper');
+    }
+
+    function it_is_a_Symfony_Templating_helper()
+    {
+        $this->shouldImplement(HelperInterface::class);
+    }
+    
+    function it_extends_base_Symfony_Templating_helper()
+    {
+        $this->shouldHaveType(Helper::class);
+    }
+
+    function it_uses_grid_renderer_to_render_grid(GridRendererInterface $gridRenderer, GridView $gridView)
```

Comment:
"Proxies" suggests Proxy pattern, which is not true here, this is just delegation.